js/bitmax.js

```diff
@@ -1122,35 +1361,32 @@ module.exports = class bitmax extends Exchange {
     }
 
     sign (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
-        let url = '/api/' + this.version + '/' + this.implodeParams (path, params);
+        let url = '/api/pro/' + this.version + '/' + this.implodeParams (path, params);
```

Comment:
This `sign()` method does not support this endpoint:

- https://bitmax-exchange.github.io/bitmax-futures-api-doc/#wallet-transaction-history
- `GET /api/pro/wallet/transactions`

Reply:
Still does not work? It should work for 'wallet/transactions'